curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f2b6fd54ac1fc142804dd.md

```diff
@@ -12,7 +12,7 @@ The logic for formatting this pyramid is likely going to get complicated, which
 A <dfn>function</dfn> is a block of code that can be reused throughout your application. Functions are declared with the following syntax:
 
 ```js
-function name(parameter) {
+function name(parameter1 , parameter2) {
```

Comment:
You need to change the text above this example, as discussed in the issue

Reply:
Updated the text